src/wasm-binary.h

```diff
@@ -1378,7 +1375,7 @@ class WasmBinaryWriter {
 
   // Writes an arbitrary heap type, which may be indexed or one of the
   // basic types like funcref.
-  void writeHeapType(HeapType type);
+  void writeHeapType(HeapType type, Exactness exact);
```

Comment:
Should we have a default of inexact to save typing?

Reply:
Since this API is local to the binary writer, I think it's best to be explicit and avoid future mistakes.